countries.css

```diff
@@ -0,0 +1,54 @@
+@import url(index.css);
+
+body {
+    background-color: var(--primary-color) ;
+  display: grid;
+   align-items: center;
+   place-items: center;
+
+}
+
+.main-container {
+    width: 500px;
+    display: grid;
+    place-items: center;
+    align-items: center;
+}
+
+.main-container .scoreboard {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 300px;
+    color: var(--secondary-color);
+    padding: 0px;
+    margin: 0px;
+    font-size: 1.3rem;
+    white-space: nowrap;
+}
+
+.question-container  {
+    margin-top: 1rem;
+    width: 100%;
+    font-size: 1.5rem;
+}
+
+.quizHeading {
+    top: 0px !important;
+    margin-top: 0px !important;
+}
+
+.answer-options {
+    text-align: center;
+    display: grid;
+    place-items: center;
+    margin-top: 2rem;
+}
+
+.answer-option {
+    background-color: var(--primary-shade-color);
+    width: 100%;
+    padding: 10px;
+    margin-bottom: 2rem;
+    border-radius: 1rem;
+    font-size: 20px;

```

Comment:
Try to use relative units (preferably rem) here to make it scale as per users' screen size or view area. It makes for a good user experience.